components/Reviews/index.jsx

```diff
@@ -1,42 +1,44 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import appConfig from '@shopgate/pwa-common/helpers/config';
-import connect from './connector';
-import styles from './style';
+import Portal from '@shopgate/pwa-common/components/Portal';
+import {
+  PRODUCT_REVIEWS,
+  PRODUCT_REVIEWS_AFTER,
+  PRODUCT_REVIEWS_BEFORE,
+} from '@shopgate/pwa-common-commerce/product/constants/Portals';
 import List from './components/List';
 import Header from './components/Header';
 import AllReviewsLink from './components/AllReviewsLink';
+import styles from './style';
 
 /**
- * Reviews Component
  * @param {Object} rating Rating of the product.
  * @param {Array} reviews Reviews which should be shown in the product page.
- * @returns {JSX|null}
+ * @returns {JSX}
  */
-const Reviews = ({ productId, rating, reviews }) => {
-  if (!appConfig.hasReviews) {
-    return null;
-  }
-
-  return (
-    <div className={styles.container} data-test-id="reviewSection">
-      <Header productId={productId} rating={rating} />
-      <List reviews={reviews} />
-      <AllReviewsLink productId={productId} />
-    </div>
-  );
-};
+const Reviews = ({ productId }) => (
+  <Fragment>
+    <Portal name={PRODUCT_REVIEWS_BEFORE} />
+    <Portal name={PRODUCT_REVIEWS}>
+      {appConfig.hasReviews && (
+        <div className={styles.container} data-test-id="reviewSection">
+          <Header productId={productId} />
+          <List productId={productId} />
```

Comment:
The `List` doesn't show reviews when it's used within the `Reviews` page

Reply:
Done